DuckDuckGo/MainViewController+Segues.swift

```diff
@@ -31,13 +31,19 @@ extension MainViewController {
     func segueToDaxOnboarding() {
         os_log(#function, log: .generalLog, type: .debug)
         hideAllHighlightsIfNeeded()
-        let storyboard = UIStoryboard(name: "DaxOnboarding", bundle: nil)
-        guard let controller = storyboard.instantiateInitialViewController(creator: { coder in
-            DaxOnboardingViewController(coder: coder)
-        }) else {
-            assertionFailure()
-            return
+
+        let controller: Onboarding?
+        
+        if DefaultVariantManager().isSupported(feature: .newOnboardingIntro) {
```

Comment:
I could inject a factory/provider at init time and have this cleaner, but since this code is something we will remove in the future, I thought it was not worth doing.